### ISSUE TITLE
The model overfits and performs poorly

### ISSUE THREAD

**Issue by mnishant2** [fastforwardlabs/extractive-summarization-with-sentencebert#2]
I tried to reproduce this model on the CNN model and train on tweetsumm data for extractive summarization, but it overfits basically from the first epoch, and validation loss is always monotonically increasing. I experimented extensively with the hyperparameters and even changed the model, but this stayed the same. Could you tell me if you also got similar performance, and if so, is there something wrong with the training regime? I have attached a picture of the validation loss of the model reproduced ditto from your notebook
<img width="601" alt="Screenshot 2022-12-24 at 12 27 15 PM" src="https://user-images.githubusercontent.com/28703440/209850197-772bbfbb-de9a-4685-a2e3-56e778239e84.png">
